t/43-dbtest.t

```diff
@@ -5,7 +5,7 @@ if (!defined $ENV{LSMB_TEST_DB}){
         plan skip_all => 'Skipping all.  Told not to test db.';
 }
 else {
-        plan tests => 470;
+        plan tests => 465;
```

Comment:
I used a number big enough to lower collisions possibilities but we'll have to think more carefully if we want to use the same method for session ids